code/index.html

```diff
@@ -3,45 +3,133 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Guess WHO?!</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="icon" type="image/x-icon" href="./images/favicon.ico">
   </head>
-  <body>
+  <body class="body">
+    <!-- The first page when the page is loaded -->
```

Comment:
It was relly clever to have like a welcome page to the game! I think it makes the impression of the game a bit more serious and well-created. 👏